.github/ci/griffin-study-utils.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+import json
```

Comment:
Is there usually a newline between line 1 and 2 here?

Reply:
`autopep8` is not adding any, and linter is happy. So I don't think so